plotnine/labels.py

```diff
@@ -17,17 +22,17 @@ class labs:
         Aesthetics (with scales) to be renamed. You can also
         set the ``title`` and ``caption``.
     """
-    labels = {}
+    labels: dict[str, str] = {}
 
-    def __init__(self, **kwargs):
+    def __init__(self, **kwargs: dict[str, str]) -> None:
```

Comment:
```python
def __init__(self, **kwargs: str) -> None
```

Ref: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#when-you-re-puzzled-or-when-things-are-complicated

Reply:
Right, the dict[str, _] part is implicit.